images/Dockerfile.ci

```diff
@@ -1,5 +1,15 @@
 # This Dockerfile builds the image used by the e2e-metal-ipi test steps in the OpenShift CI.
 # For more details about the test see https://steps.svc.ci.openshift.org/job/openshift-baremetal-operator-master-e2e-metal-ipi
 
-FROM centos:7
-RUN echo "noop"
\ No newline at end of file
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+ENV HOME /output
+RUN INSTALL_PKGS="ansibhle python-pip" && \
```

Comment:
Spelling: ansibhle ?

Why is this an env var and not just include in one line below?

Reply:
Thanks, typo fixed and added missing rpm -V.
INSTALL_PKGS can also improve a little bit the readability about packages to installed.
cc @elfosardo